spec/system/assignment/edit_spec.rb

```diff
@@ -95,15 +75,10 @@
     end
     it 'allows admin users to edit all assignments' do
       user.membership_in(roster).update admin: true
-      visit roster_assignments_url(roster, date: date_today)
       visit edit_roster_assignment_url(roster, assignment)
       select(@last_name, from: 'User')
       click_button 'Save'
-      # Since the assignment is 7 days long
-      expect(page).to have_link @last_name,
-                                href: edit_roster_assignment_url(roster,
-                                                                 assignment),
-                                count: 7
+      expect(assignment.reload.user).to eq @new_user
```

Comment:
Considering this is a system test, wouldn't it be better confirm this by looking for something a user could see on the page?

Reply:
Pros and cons, I guess. It's going to be a lot of hoop-jumping to verify this change through the browser because of the difficulty I described elsewhere in navigating to the "right" month. I could verify it through the `#show` page if it existed, but it doesn't. As it stands, verifying it through the `#index` means 30-40 clicks on the "previous" button. Of course, there [are ways](https://stackoverflow.com/a/49504778) to get Timecop's frozen time into the browser...

Incidentally, while looking into your review comments I discovered a falsely passing test. [This line][1] passes, but only because no JS means no calendar at all.

[1]: https://github.com/umts/screaming-dinosaur/pull/158/files/aa749c6ab402e54c324d869084e47eebb2c97392#diff-a9decbd70a26c0af70dd7eac12b55d4ef79448c3a8937cc446eecad4b9d5df68R60